src/main.ts

```diff
@@ -233,3 +236,13 @@ ipcMain.handle('set_main_window_title', (_event, title: string) => {
   const mainWindow = BrowserWindow.getAllWindows()[0];
   mainWindow?.setTitle(title);
 });
+
+ipcMain.handle('check_for_updates', async (_event) => {
+  const response = await fetch('https://api.github.com/repos/diffix/desktop/releases?per_page=1');
```

Comment:
We should ignore prereleases from notifying the user.

Reply:
settled on:

> edon  15 minutes ago
> so to summarize: fetch /latest, if 404 return null, else compare versions
> piotr  14 minutes ago
>  \+ OK to crash on anything else, as long as the app lives